feature-libs/product/configuration/src/textfield-configurator/facade/configurator-textfield.service.ts

```diff
@@ -27,32 +29,37 @@ export class ConfiguratorTextfieldService {
    *
    * @returns {Observable<ConfiguratorTextfield.Configuration>}
    */
-  public createConfiguration(
+  createConfiguration(
     owner: GenericConfigurator.Owner
   ): Observable<ConfiguratorTextfield.Configuration> {
     this.store.dispatch(
```

Comment:
I would be happy to have `this.store.dispatch` as part of the same stream as `this.store.pipe` (in a `tap`).
Is there a reason why it's not part of it? Perhaps you had an infinite loop, or something similar?

Reply:
If this is the case, then it should not be a problem to refactor this, as you are using the _LoaderState_ and you have access to _success_ and _loading_ flags.

Reply:
To be honest here we didn't consider this (while for the other configurator we did). Let me try to refactor.